src/main/kotlin/org/rust/ide/hints/RsGenericParameterInfoHandler.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.hints
+
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.lang.parameterInfo.*
+import com.intellij.openapi.util.TextRange
+import com.intellij.util.containers.nullize
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+
+private const val WHERE_PREFIX = "where "
+
+class RsGenericParameterInfoHandler : ParameterInfoHandler<RsTypeArgumentList, RsGenericPresentation> {
+
+    var curParam = -1
+        private set
+    var hintText = ""
+        private set
+
+    override fun showParameterInfo(element: RsTypeArgumentList, context: CreateParameterInfoContext) {
+        context.highlightedElement = null
+        context.showHint(element, element.textRange.startOffset, this)
+    }
+    
```

Comment:
Forgotten 4 spaces on this line. I'd recommend enabling the feature to auto remove redundant tabulation :)